### DISCUSSION TITLE
Create a video walkthrough?

### DISCUSSION THREAD

**Discussion by ttyrawa** [kudrykv/latex-yearly-planner#55] [General]
Hey so I see in the readme that you'd like to create documentation on how one would be able to create their own templates using your method. And I totally understand how long that would take to do. 

I was wondering if you would maybe be able to create a video if it is faster, just explaining the flow of the go app and how it builds the pdfs based off your tex files. 

Or if you have the time or don't like want to make a video, if you could write up a high level overview flow of how this program works, so someone could tweak the tex template files and create a different style planner that would still use code to auto generate each month,week, ect

**Comment by kudrykv** [comment]
Hi @ttyrawa!

Yeah, that makes sense. I might record something up quickly. Thanks for the suggestion!
Here is a high-level overview of how `main` branch works today: https://github.com/kudrykv/latex-yearly-planner/discussions/34#discussioncomment-3128344

**Reply by kudrykv** [reply]
`single.sh` is the entry point. You can unroll from there.

Roughly:
1. Go code generates tex files.
2. pdflatex creates PDF from tex files.

In turn, Go uses tex snippets from `tpls` dir
